configure.scss

```diff
@@ -11,6 +11,7 @@ $_n-ui-features: (
 	grid: true,
 	header: true,
 	icons: null,
+	messages: true,
```

Comment:
should be `false` here, otherwise they would get initialised on error pages, opt out form etc